RecoParticleFlow/PFTracking/interface/PFTrackTransformer.h

```diff
@@ -58,6 +58,7 @@ class PFTrackTransformer {
   ///B field
   math::XYZVector B_;
   PFGeometry pfGeometry_;
+  double cos2ThetaV_Endcap_HiEnd_;
```

Comment:
Defining here as ```static constexpr double``` could be a better choice

Reply:
Tried it now.